cdp_scrapers/instances/seattle.py

```diff
@@ -18,6 +22,26 @@
 
 ###############################################################################
 
+STATIC_FILE_KEY_PERSONS = "persons"
+STATIC_FILE_DEFAULT_PATH = "cdp_scrapers/instances/seattle-static.json"
```

Comment:
Should change to `Path(__file__).parent / "seattle-static.json"`

`Path` is from `pathlib`, `__file__` tells python to grab the file this code is from. So `Path(__file__).parent` says "current directory" then get the related file.

Also you should include this file / all JSON in the MANIFEST.in -- see [cdp-backend](https://github.com/CouncilDataProject/cdp-backend/blob/main/MANIFEST.in)

Reply:
> `Path` is from `pathlib`, `__file__` tells python to grab the file this code is from. So `Path(__file__).parent` says "current directory" then get the related file.

Thank you for the tips as always. :raised_hands: